components/micro-gateway-core/src/main/ballerina/gateway/listeners/api_gateway_listener.bal

```diff
@@ -26,25 +26,31 @@ import ballerina/io;
 public type APIGatewayListener object {
     *AbstractListener;
 
+    private int listenerPort = 0;
+    private string listenerType = "HTTP";
     public http:Listener httpListener;
 
-
     public function __init(int port, http:ServiceEndpointConfiguration config) {
-        int port1 = 0;
         if ((config.secureSocket is ())) {
-            port1 = getConfigIntValue(LISTENER_CONF_INSTANCE_ID, LISTENER_CONF_HTTP_PORT, port);
+            self.listenerPort = getConfigIntValue(LISTENER_CONF_INSTANCE_ID, LISTENER_CONF_HTTP_PORT, port);
         } else {
-            port1 = getConfigIntValue(LISTENER_CONF_INSTANCE_ID, LISTENER_CONF_HTTPS_PORT, port);
+            self.listenerPort = getConfigIntValue(LISTENER_CONF_INSTANCE_ID, LISTENER_CONF_HTTPS_PORT, port);
+            self.listenerType = "HTTPS";
         }
         initiateGatewayConfigurations(config);
-        printDebug(KEY_GW_LISTNER, "Initialized gateway configurations for port:" + port1);
-        self.httpListener = new(port1, config = config);
-        printDebug(KEY_GW_LISTNER, "Successfully initialized APIGatewayListener for port:" + port1);
+        printDebug(KEY_GW_LISTNER, "Initialized gateway configurations for port:" + self.listenerPort);
+
+        self.httpListener = new(self.listenerPort, config = config);
+
+        printDebug(KEY_GW_LISTNER, "Successfully initialized APIGatewayListener for port:" + self.listenerPort);
     }
 
 
     public function __start() returns error? {
-        return self.httpListener.__start();
+        error? gwListener = self.httpListener.__start();
+
+        io:println("[micro-gateway] " + self.listenerType + " listener active on port " + self.listenerPort);
```

Comment:
Lets use log:info()

Reply:
@Rajith90 Use of log results in an output like this.
```ballerina: HTTP access log enabled
[ballerina/http] started HTTPS/WSS endpoint 0.0.0.0:9096
[ballerina/http] started HTTPS/WSS endpoint 0.0.0.0:9095
[ballerina/http] started HTTP/WS endpoint 0.0.0.0:9090
2019-05-16 21:43:12,411 INFO  [wso2/gateway] - [micro-gateway] HTTPS listener is active on port 9095 
2019-05-16 21:43:12,411 INFO  [wso2/gateway] - [micro-gateway] HTTP listener is active on port 9090
```
That doesn't look nice